docs/Targets.md

```diff
@@ -1,47 +1,44 @@
-# Targets
-
-This README describes configuration of supported targets.
-
-## Supported Targets
-
-* [Simulated](#simulated)
-* [Cortex-A53 / Raspberry PI 3](#cortex-a53--raspberry-pi-3-experimental)
-* [Cypress PSoC-6](#cypress-psoc-6)
-* [Infineon AURIX TC3xx](#infineon-aurix-tc3xx)
-* [Intel x86-64 Intel FSP](#intel-x86_64-with-intel-fsp-support)
-* [Kontron VX3060-S2](#kontron-vx3060-s2)
-* [Microchip SAMA5D3](#microchip-sama5d3)
-* [Microchip SAME51](#microchip-same51)
-* [Nordic nRF52840](#nordic-nrf52840)
-* [Nordic nRF5340](#nordic-nrf5340)
-* [NXP iMX-RT](#nxp-imx-rt)
-* [NXP Kinetis](#nxp-kinetis)
-* [NXP LPC54xxx](#nxp-lpc54xxx)
-* [NXP LS1028A](#nxp-ls1028a)
-* [NXP MCXA153](#nxp-mcxa153)
-* [NXP P1021 PPC](#nxp-qoriq-p1021-ppc)
-* [NXP T1024 PPC](#nxp-qoriq-t1024-ppc)
-* [NXP T2080 PPC](#nxp-qoriq-t2080-ppc)
-* [Qemu x86-64 UEFI](#qemu-x86-64-uefi)
-* [Raspberry Pi pico 2 (rp2350)](#raspberry-pi-pico-rp2350)
-* [Renesas RA6M4](#renesas-ra6m4)
-* [Renesas RX65N](#renesas-rx65n)
-* [Renesas RX72N](#renesas-rx72n)
-* [Renesas RZN2L](#renesas-rzn2l)
-* [SiFive HiFive1 RISC-V](#sifive-hifive1-risc-v)
-* [STM32C0](#stm32c0)
-* [STM32F4](#stm32f4)
-* [STM32F7](#stm32f7)
-* [STM32G0](#stm32g0)
-* [STM32H5](#stm32h5)
-* [STM32H7](#stm32h7)
-* [STM32L0](#stm32l0)
-* [STM32L4](#stm32l4)
-* [STM32L5](#stm32l5)
-* [STM32U5](#stm32u5)
-* [STM32WB55](#stm32wb55)
-* [TI Hercules TMS570LC435](#ti-hercules-tms570lc435)
-* [Xilinx Zynq UltraScale](#xilinx-zynq-ultrascale)
+# Target Configuration Guide
+
+This guide provides configuration and setup instructions for all supported targets in wolfBoot.
+
+## Quick Reference
+
+### Target Categories
+
+#### ARM-based MCUs
+- STM32 Family (F4, F7, G0, H5, H7, L0, L4, L5, U5, WB55, C0)
+- NXP Family (iMX-RT, Kinetis, LPC54xxx, MCXA153)
+- Nordic Family (nRF52840, nRF5340)
+- Renesas Family (RA6M4, RX65N, RX72N, RZN2L)
+- Other ARM (Cypress PSoC-6, Microchip SAME51)
+
+#### RISC-V
+- SiFive HiFive1
+
+#### PowerPC
+- NXP QorIQ (P1021, T1024, T2080)
+
+#### x86-64
+- Intel FSP Support
+- QEMU x86-64 UEFI
+- Kontron VX3060-S2
+
+#### Development/Testing
+- Simulated Target
+- QEMU Support
+
+## Target Index
+
+| Target | Architecture | Key Features | Documentation Link |
+|--------|--------------|--------------|-------------------|
+| STM32F4 | ARM Cortex-M4 | Dual Bank Flash, TrustZone | [Details](#stm32f4) |
```

Comment:
The STM32F4 does not support TrustZone, and only a limited few models support dual-bank flash.

See https://www.st.com/en/microcontrollers-microprocessors/stm32f4-series.html